lib/ansible/modules/find.py

```diff
@@ -436,6 +436,8 @@ def main():
                         wpath = npath.rstrip(os.path.sep) + os.path.sep
                         depth = int(fsname.count(os.path.sep)) - int(wpath.count(os.path.sep)) + 1
                         if depth > params['depth']:
+                            # Empty the list used by os.walk to avoid traversing deeper unnecessarily
+                            del(dirs[:])
```

Comment:
[Python documentation](https://docs.python.org/3/tutorial/datastructures.html) says that `del dirs[:]` is equivalent to `dirs.clear()`. Just out of curiosity is there a reason to use the former? Is it faster maybe?

Reply:
`clear()` does not seem to exist in py2.7